src/rgw/rgw_rest.cc

```diff
@@ -1668,6 +1668,13 @@ int RGWDeleteMultiObj_ObjStore::get_params(optional_yield y)
   return op_ret;
 }
 
+int RGWStatBucket_ObjStore::get_params(optional_yield y)
+{
+  report_stats = s->cct->_conf->rgw_bucket_stats_on_head_bucket &&
+                 s->info.env->exists("HTTP_X_RGW_GET_BUCKET_STATS");
```

Comment:
we'll want to announce this change in the next release. can you please add a sentence about it to PendingReleaseNotes?

Reply:
I added this to the section for the version 20. Should I also/only add this to version 19 for the backport reasoning?